.github/workflows/stale_issues.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. We might have no time to process reports for websites that are not popular. It will be closed if no further activity occurs. However, if you are interested to solve the issue, you can create a pull request following [these instructions](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md). Thank you for your contributions.'
```

Comment:
better with an exact date
`has not had recent activity` => `has been open ** days with no activity` or smth. like that.